src/pages/video-test.tsx

```diff
@@ -36,56 +36,107 @@ export const getServerSideProps: GetServerSideProps = async function ({query}) {
   }
 }
 
-const TestSidePanel: React.FC<any> = ({className, disableCompletely}) => {
+function getNoteCues() {
```

Comment:
this should be `useNoteCues` and follow React naming conventions as a hook since it follows that pattern and actually utilizes a hook internally.